spec/lib/dfid/patch/countries_spec.rb

```diff
@@ -58,10 +58,10 @@ def page(index)
 
         before do
           allow(RestClient).to receive(:get).with(
```

Comment:
It was probably reasonable to stub these before, but now we're effectively stubbing implementation details for `GovUkCountryRegister`, which could change (perhaps we change to `Net::HTTP`?). That said, the alternative is to return a bunch of country register-specific JSON loaded directly from two pages of `fixtures/service-results/.json`.